lib/tasks/sendgrid/init.rake

```diff
@@ -22,15 +22,15 @@ namespace :sendgrid do
       end
 
       # Publisher's List
-      publishers_list = SendGrid::ApiHelper.get_list(list_name: 'Brave Payments')
+      publishers_list = SendGrid::ApiHelper.get_list(list_name: 'Brave Rewards')
```

Comment:
I think we need to keep this as Brave Payments since this is a value that's on sendgrid. Maybe we can change it there

Reply:
I changed it there on SendGrid 💃